packages/protocol/contracts/governance/Governance.sol

```diff
@@ -754,7 +756,7 @@ contract Governance is
     require(value > 0, "Nothing to withdraw");
     require(value <= address(this).balance, "Inconsistent balance");
     refundedDeposits[msg.sender] = 0;
-    msg.sender.transfer(value);
+    msg.sender.sendValue(value);
```

Comment:
do we need to use the `toPayable` in collaboration with this?
https://docs.openzeppelin.com/contracts/2.x/api/utils#Address-toPayable-address-

Reply:
I wonder if this is related to the prettier issue

Reply:
the docs also flag taking care not to introduce reentrancy issues

Reply:
I was thinking the prettier issue may have had to do with the solidity linter linting an earlier version of solidity before there was a separate `address payable` type....
`msg.sender` is inherently `address payable` so I don't think so.....? I think it would have failed to compile

will take a second look at the reentrancy stuff